path_role.go

```diff
@@ -70,6 +70,18 @@ should never expire. The token should be renewed within the
 duration specified by this value. At each renewal, the token's
 TTL will be set to the value of this parameter.`,
 			},
+			"expiration_leeway": {
+				Type: framework.TypeDurationSecond,
+				Description: `Duration in seconds of leeway when validating expiration of a token to account for clock skew. 
+Defaults to 300 (5 minutes).`,
+				Default: "300",
```

Comment:
Probably want to avoid defaulting here and for `not_before_leeway` so we don't persist this value (which in turn let's us change it for those who haven't explicitly set it).

Reply:
I think this is a good idea but it makes it a little awkward because the role configuration says one thing (IE leeways are 0) but logins do another thing (default to 300 seconds).  Do you think this might be confusing to users?

Reply:
@kalafut Added your suggestion for review.